src/tasks/processIssueComment/Repository/index.js

```diff
@@ -106,7 +106,7 @@ class Repository {
             owner: this.owner,
             repo: this.repo,
             path: filePath,
-            message: `docs: update ${filePath}`,
+            message: `docs: update ${filePath} ${this.skipCi}`,
```

Comment:
I think `this.skipCi` can be undefined, which would result in `undefined`. You’d have to make sure it’s a string when setting this.skipCi or add a ` || ''` fallback here

Reply:
As I stated before I know like nothing about javascript, even thougth I did a few research for this PR.
But IMHO, the message are processed after
``` js
        repository.skipCi = optionsConfig.options.skipCi
```
in the file below, so it should be defined.
Maybe improvments could come from a better test, as you said, something like
``` js
if (typeof this.options.skipCi !== 'string') {
    optionsConfig.options.skipCi = '[skip ci]'
}
```
in the optionsConfig ensure valid function.
I think we should also add the skipCi string to the 'create all-contributorsrc' message.
What do you think?

> According to my tests, this would work, but I might be missing something.

Reply:
I’m not familiar enough with the code base yet, but will look into it soon. Ideally we’d add a test. I don’t feel confident enough to merge your PR yet, but someone else will beat me to it :)

Reply:
Unless you plan on adding a `skipCi` setting in `Repository.constructor`, using `${this.skipCi || ''}` would be a must as pointed out by @gr2m.

Reply:
``` js
${this.skipCi || ''}
```
The syntax is very nice!
But as there are two places to use it, I think it is better to do it in the constructor.

Reply:
> But as there are two places to use it, I think it is better to do it in the constructor.

Good point.